docker-compose.yml

```diff
@@ -0,0 +1,102 @@
+version: "3.7"
```

Comment:
A bit everywhere in this file you're using default values for confidentials or configurable information (like password, api keys etc.)

You're expecting the user to edit this file to secure his instance of build-raven-os-org. The problem is that it slows down the update process, requiring the user to modify this file everytime. 

A better approach would be to use environment variables instead of hardcoded values, so the user can git pull the project and run it again (using the same environment variables) in a few seconds.